crates/core/Cargo.toml

```diff
@@ -29,6 +29,8 @@ hex.workspace = true
 itertools.workspace = true
 futures-util = "0.3.30"
 m3u8-rs.workspace = true
+sha2 = "0.10.8"
+bech32 = "0.11.0"
```

Comment:
add these to the workspace dependencies and update zap-stream crate to use sha2 from workspace